src/Chainweb/Pact/Backend/Orphans.hs

```diff
@@ -298,23 +267,24 @@ instance Serial1 Def where
         _dDefBody <- deserializeWith m
         _dMeta <- deserialize
         _dInfo <- deserialize
-        return $ Def {..}
+        return $ Def{..}
 
+-- FIXME: This should be autoderivable, since `Generic1` is autoderivable.
```

Comment:
I'm just leaving this note as a reminder of our discussion in which we agreed upon there needing to be a separate PR to replace these instances with the auto-derivable ones.